### PR TITLE
chore: `useContext` in Empty

### PR DESCRIPTION
**Pull request by Amour1688**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | use `useContext` instead of `ConfigConsumer` in Empty  |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/empty/index.tsx [hunk 1]
```diff
@@ -30,59 +30,56 @@ interface EmptyType extends React.FC<EmptyProps> {
...
+  const { getPrefixCls, direction } = React.useContext(ConfigContext);
+  const {
+    className,
+    prefixCls: customizePrefixCls,
+    image = defaultEmptyImg,
+    description,
+    children,
+    imageStyle,
+    ...restProps
+  } = props;
```
Comment (hengkx) [inline components/empty/index.tsx:43]:
```suggestion
const Empty: EmptyType = ({
    className,
    prefixCls: customizePrefixCls,
    image = defaultEmptyImg,
    description,
    children,
    imageStyle,
    ...restProps
  }) => {
  const { getPrefixCls, direction } = React.useContext(ConfigContext);
```

**Comment by afc163** [comment]
请补充这部分的内容：

image

**Comment by Amour1688** [comment]
> 请补充这部分的内容：
> 
> image

done

**Comment by afc163** [comment]
⚠️ Please check all items below before review. ⚠️

**Pull request merged**